### PR TITLE
[Communication] - Identity - Added managed identity tests and updated README

### PR DESCRIPTION
**Pull request by jbeauregardb**

### CODE REVIEW

File: sdk/communication/azure-communication-administration/README.md [hunk 1]
```diff
@@ -28,15 +28,39 @@ Acquired phone numbers can come with many capabilities, depending on the country
 
 ## Key concepts
 
-To use the Admnistration SDK, a resource access key is required for authentication. 
+There are two forms of authentication to use the Admnistration SDK.
 
+### Using your AAD Credentials
```
Comment (JianpingChen) [inline sdk/communication/azure-communication-administration/README.md:33]: To make the page more comprehensive, this header should read similar to the one below, "Access Key Authentication", e.g. "Active Directory Token Authentication"

**Review by JianpingChen — CHANGES_REQUESTED** [review]

**Review by JianpingChen — APPROVED** [review]

**Comment by minnieliu** [comment]
I would suggest running the Live test pipeline on the new tests before merging your PR in

File: sdk/communication/azure-communication-administration/README.md [hunk 2]
```diff
@@ -28,15 +28,40 @@ Acquired phone numbers can come with many capabilities, depending on the country
 
 ## Key concepts
 
-To use the Admnistration SDK, a resource access key is required for authentication. 
+There are two forms of authentication to use the Administration SDK:
 
+### Azure Active Directory Token Authentication
+Currently, Azure Active Directory (AAD) authentication is only possible for CommunicationIdentityClient.
+The DefaultAzureCredential object must be passed to the CommunicationIdentityClientBuilder via
```
Comment (minnieliu) [inline sdk/communication/azure-communication-administration/README.md:35]: Code format for `DefaultAzureCredential `?

Comment (minnieliu) [inline sdk/communication/azure-communication-administration/README.md:35]: Also Code format for CommunicationIdentityClientBuilder

File: sdk/communication/azure-communication-administration/README.md [hunk 3]
```diff
@@ -28,15 +28,40 @@ Acquired phone numbers can come with many capabilities, depending on the country
...
+the credential() funtion. Endpoint and httpClient must also be set
+via the endpoint() and httpClient() functions respectively.
+
+CLIENT_SECRET, CLIENT_ID and TENANT_ID environment variables
```
Comment (minnieliu) [inline sdk/communication/azure-communication-administration/README.md:39]: Here as well, code formatting

Comment (minnieliu) [inline sdk/communication/azure-communication-administration/README.md:39]: environment variables **are** needed to create..

**Review by minnieliu — APPROVED** [review]

**Pull request merged**